### ISSUE TITLE
calimero dimming read and write

### ISSUE THREAD

**Issue by ViorelOnica** [calimero-project/calimero-core#64]
Hello,
I have 2 problems:
1) Reading switch (1bit boolean) and dimming (1byte unsigned int) datapoints. 
In order to do that, I used these 2 functions below, but "catch" occurs.

![read](https://user-images.githubusercontent.com/27728770/39675568-8cbcb326-5165-11e8-82ab-bcc260297f36.png)

Below you can see the objects I created and the logs read attempts.
![issue](https://user-images.githubusercontent.com/27728770/39675576-b3f359fe-5165-11e8-9483-16555f216d00.png)

Is it about bad objects configuration or am I using wrong the library methods ?

2) I have a method that gets triggered whenever knx connection gets lost:
@Override
public void linkClosed(CloseEvent e) { //code to execute }

But I tried the following scenario: turned off mobile data and this function gets triggered after 30-45 seconds after losing the internet connection. 
You told me in another issue post that what I'm seeking for is an idle state trigger over the knxlink.. I failed finding the proper function that takes care of that, can you give me a concrete code example, please ?

I'd really appreciate your help !

**Comment by calimero-project** [comment]
The way you use the methods is fine.

What exactly is the error (output of `e.printStackTrace()`)?

Re link: if your goal is to automatically reconnect and/or configure that in more detail, you can use Connector (`tuwien.auto.calimero.link.Connector`). Even if the network connection was down (and/or the link got closed), it will re-establish a link.

**Comment by ViorelOnica** [comment]
![exc](https://user-images.githubusercontent.com/27728770/39769627-6229e6ba-52f5-11e8-8422-85e67443f3e5.png)

This exception is threw after calling any of the 2 functions (readBoolean, readDimming) ..;\

Concerning the knxlink: 
Yes, I want to automatically reconnect, but there's a green circle in the application indicating that the connection's up and I want to make it red whenever it goes idle ( I'm using the linkClosed method and it's getting red only after 30-45 seconds after losing the mobile data )

**Comment by calimero-project** [comment]
This exception means that the remote endpoint (the one that should answer for that group address) is not answering, therefore, a timeout exception is thrown after the specified process communication timeout.

**Comment by ViorelOnica** [comment]
hmm, okay, thanks.. I'll do some research

**Comment by ViorelOnica** [comment]
well, I'm not getting timeout exception anymore and I'm getting a response aswell

![eff](https://user-images.githubusercontent.com/27728770/39868072-ae944916-5460-11e8-83cc-156d03e43759.png)

But this is the new exception I get

![ff](https://user-images.githubusercontent.com/27728770/39868026-702061e2-5460-11e8-960f-05dfdb9f67fe.png)
